docs/reference/js/api.md

```diff
@@ -29,10 +32,18 @@ If you are writing an application that needs to access the IPFS network, use `ip
 
 :::
 
-### HTTP client
+### HTTP clients
+
+[Kubo HTTP RPC API](https://www.npmjs.com/package/kubo-rpc-client) is a client library that controls an active Kubo node running through its [RPC API](../kubo/rpc.md).
+  - Use this library to talk to a Kubo node from your JavaScript application
+  - You can also interact with the [RPC API](../kubo/rpc.md) directly using `fetch()` in a browser or a module like `request` in Node.js, but using this library can be much more convenient.
+
+::: warning Changes to IPFS HTTP RPC API
 
-[JS-IPFS HTTP RPC API](https://www.npmjs.com/package/ipfs-http-client) is a client library that controls an active IPFS node (Kubo or JS-IPFS) running through its [RPC API](../kubo/rpc.md).
-  - JS-IPFS will internally use this library if it detects another node is running on your machine. You can also interact with the [RPC API](../kubo/rpc.md) directly using `fetch()` in a browser or a module like `request` in Node.js, but using this library can be much more convenient.
+There is ongoing work that will result in HTTP RPC API divergence in the future. See https://github.com/ipfs/kubo/issues/9125 for more information.
+
+[JS-IPFS HTTP RPC API](https://www.npmjs.com/package/ipfs-http-client) is a client library that controls an active IPFS node (JS-IPFS) running through its [RPC API](../kubo/rpc.md).
+  - JS-IPFS will internally use this library if it detects another node is running on your machine. You can also interact with the [RPC API](../kubo/rpc.md) directly using `fetch()` in a browser or a module like `request` in Node.js, but using this library can be much more convenient. JS-IPFS implements it's [own version of the RPC API](https://github.com/ipfs/js-ipfs/tree/master/packages/ipfs-http-server).
```

Comment:
Let's not link to Kubo docs, it's confusing. 
JS-IPFS has both HTTP and gRPC, let's link to that.

```suggestion
JS-IPFS [ipfs-http-client](https://www.npmjs.com/package/ipfs-http-client) is a library that controls an active IPFS node (JS-IPFS) running through its own version of the [RPC API over HTTP](https://github.com/ipfs/js-ipfs/tree/master/packages/ipfs-http-server).
  - JS-IPFS will internally use this library if it detects another node is running on your machine. You can also interact with the HTTP version of JS-IPFS RPC API directly using `fetch()` in a browser or a module like `request` in Node.js, but using this library can be much more convenient. 
```